test/e2e/play_kube_test.go

```diff
@@ -6420,4 +6420,13 @@ spec:
 
 		Expect(testfile).ToNot(BeAnExistingFile(), "file should never be created on the host")
 	})
+
+	It("test container name without Pod name prefix", func() {
+		err := writeYaml(podnameEqualsContainerNameYaml, kubeYaml)
+		Expect(err).ToNot(HaveOccurred())
+
+		podmanTest.PodmanExitCleanly("kube", "play", "--no-pod-prefix", kubeYaml)
```

Comment:
The same pod name is used in other tests. I am guessing there is a problem with the teardown of the tests.
`Error: name "podnameEqualsContainerNameYaml" is in use: pod already exists`

Reply:
Yeah, I noticed that. Will fix soon.